projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-google-docs.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Validate whether a google doc is available for embedding.
+ *
+ * @package automattic/jetpack
+ */
+
+use Automattic\Jetpack\Extensions\GoogleDocsEmbed;
+
+/**
+ * Google Docs block endpoint.
+ */
+class WPCOM_REST_API_V2_Endpoint_Google_Docs extends WP_REST_Controller {
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		add_action( 'rest_api_init', array( $this, 'register_routes' ) );
+	}
+
+	/**
+	 * Register endpoint route.
+	 */
+	public function register_routes() {
+		register_rest_route(
+			'wpcom/v2',
+			'/checkDocumentVisibility',
```

Comment:
Now that we're in the `wpcom/v2` namespace, maybe the endpoint name should mention Google, so we immediately know what this is about?

Reply:
Updated.